sdk/platform/src/persistence/mongodb/analytics.rs

```diff
@@ -76,34 +71,27 @@ impl Pipeline {
         // Process the results
         match cursor.next().await {
             Some(result) => {
-
                 self.clear();
 
                 match result {
                     Ok(doc) => match serde_json::to_value(&doc) {
                         Ok(value) => Ok(value),
-                        Err(err) => Err(
-                            Error::Mongo(
-                                format!("Error serializing to Json: {}", err)
-                            )
-                        )
+                        Err(err) => {
+                            Err(Error::Mongo(format!("Error serializing to Json: {}", err)))
+                        }
                     },
-                    Err(err) => Err(
-                        Error::Mongo(
-                            format!("Error extracting document from MongoDB Result: {}", err)
-                        )
-                    )
+                    Err(err) => Err(Error::Mongo(format!(
+                        "Error extracting document from MongoDB Result: {}",
+                        err
+                    ))),
                 }
-            },
+            }
             None => {
-
                 self.clear();
 
-                Err(
-                    Error::Mongo(
-                        String::from("No result from DocumentDB Aggregate")
-                    )
-                )
+                Err(Error::Mongo(String::from(
+                    "No result from DocumentDB Aggregate",
+                )))
```

Comment:
and same here - Is this enforced with a linter rule? If not, can it be?

Reply:
See previous comment about linter and splitting large commits.